administrator/components/com_admin/sql/updates/postgresql/4.1.0-2021-08-04.sql

```diff
@@ -0,0 +1,9 @@
+-- Add "com_cronjobs" to "#__extensions"
+INSERT INTO "#__extensions" ("package_id", "name", "type", "element", "folder", "client_id", "enabled", "access",
+							 "protected", "locked", "manifest_cache", "params", "custom_data", "ordering", "state")
+VALUES (0, 'com_cronjobs', 'component', 'com_cronjobs', 1, 1, 1, 1, 0, 1, '', '{}', '', 0, 0);
```

Comment:
```suggestion
INSERT INTO "#__extensions" ("package_id", "name", "type", "element", "folder", "client_id", "enabled", "access",
							 "protected", "locked", "manifest_cache", "params", "custom_data", "ordering", "state")
VALUES (0, 'com_cronjobs', 'component', 'com_cronjobs', '', 1, 1, 1, 0, 1, '', '{}', '', 0, 0);
```
This SQL statement is wrong because the columns list includes the `folder` column, but the values list provides not an empty string like it should be but a number for that column.

Reply:
Thank you for reviewing!